### PR TITLE
Add contrib info to the readme

### PR DESCRIPTION
**Pull request by jpoirier**

### CODE REVIEW

**Comment by jpoirier** [comment]
@racerxdl @hayguen can you guys review the steps when you get a chance

**Comment by racerxdl** [comment]
Looks ok for me :+1:

File: README.md [hunk 1]
```diff
@@ -16,14 +16,16 @@ http://sdr.osmocom.org/trac/wiki/rtl-sdr
...
+- add the upstream development repo:
+    * git remote add upstream git@github.com:librtlsdr/librtlsdr.git
+- track the development branch: 
+    * git branch --track development origin/development
+- checkout the development branch and make your changes
 - commit your changes
 - sync your local development branch with the upstream development branch:
-    git fethc upstream
-    git merge upstream/development
+    * git fethc upstream
```
Comment (rct) [inline README.md]: Typo: fetch

**Comment by kerel-fs** [comment]
What is the purpose of the `master` branch, when the development happens in the `development` branch?

**Comment by racerxdl** [comment]
@kerel-fs the master branch has the stable build, the one we've tested and should be used for production. The development might have things that are broken or that doesn't work well.

**Pull request merged**